front/.eslintrc.js

```diff
@@ -16,7 +16,8 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'plugin:prettier/recommended',
     'plugin:storybook/recommended',
-    'react-app',
+    'plugin:react/recommended',
```

Comment:
I'm moving to these plugins as the react-app config is not compatible with the newer eslint version

Reply:
Alright